videozen-frontend/src/components/PlaylistVideo/CurrentVideo/CurrentVideo.jsx

```diff
@@ -0,0 +1,128 @@
+import { useData } from "../../../context/data-context";
+
+import { useState } from "react";
+import { useLocation } from "react-router-dom";
+
+import Toast from "../../Toast/Toast";
+import ModalPart from "../ModalPart/ModalPart";
+
+import AddPlaylist from "../../../assets/playlist1.svg";
+
+const CurrentVideo = (props) => {
+  const [showModal, setShowModal] = useState(false);
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState(null);
+
+  const { state: video } = useLocation();
+  const { state, dispatch } = useData();
+
+  const utilToast = (message) => {
+    setShowToast(true);
+    setToastMessage(message);
+    setTimeout(() => setShowToast(false), 2000);
+  };
+
+  const isTheVideoLiked = (videoId) => {
+    for (let i = 0; i < state.likedVideos.length; i++) {
+      if (state.likedVideos[i].id === videoId) {
+        return true;
+      }
+    }
+    return false;
+  };
+
+  const isTheVideoAddedToWatchLater = (videoId) => {
+    const watchLaterVideos = state.createdPlaylists[0].videos;
+
+    for (let i = 0; i < watchLaterVideos.length; i++) {
+      if (watchLaterVideos[i].id === videoId) {
+        return true;
+      }
+    }
+
+    return false;
+  };
+
+  const likeHandler = () => {
+    if (isTheVideoLiked(video.id)) {
+      dispatch({
+        type: "REMOVE_FROM_LIKED_VIDEOS",
+        payload: { video: video },
+      });
+    } else {
+      dispatch({ type: "ADD_TO_LIKED_VIDEOS", payload: { video: video } });
+    }
+  };
```

Comment:
Remove if-else and add return statement
```suggestion
    if (isTheVideoLiked(video.id)) {
      return dispatch({
        type: "REMOVE_FROM_LIKED_VIDEOS",
        payload: { video: video },
      });
    } 
    return dispatch({ type: "ADD_TO_LIKED_VIDEOS", payload: { video: video } });
    }
  };
```